src/current/cockroachcloud/costs.md

```diff
@@ -260,7 +260,7 @@ Cloud providers may vary in how they calculate each category and define each reg
 
 #### Same-region
 
-There are no charges for same-region data transfer between CockroachDB cluster nodes. Data egressing the cluster to endpoints in the same region for backups or changefeeds, or using private connectivity, is charged the cloud provider's list price for same-region data transfer.
+There are no charges for same-region data transfer between CockroachDB cluster nodes. Data egressing the cluster to endpoints in the same region for backups or changefeeds, or using private connectivity, is charged the cloud provider's list price for same-region data transfer. Egress private endpoint usage is posted as separate line items under _Private endpoint - bytes processed_.
```

Comment:
My understanding is the _Private Endpoint - bytes processed_  is a line item within Data Transfer and not Data Transfer -> Same Region. @vishalv1994 Can you please confirm?

Reply:
I think you're looking at an older commit, I responded to Vishal's feedback and the current text reads:

> There are no charges for same-region data transfer between CockroachDB cluster nodes. Data egressing the cluster to endpoints in the same region for backups or changefeeds, or using private connectivity, is charged the cloud provider's list price for same-region data transfer. Egress private endpoint usage is posted in separate line items under the **Data transfer** section, as  **Private endpoint - bytes processed - Advanced plan**.